src/components/AddEditMovie/AddEditMovie.jsx

```diff
@@ -1,130 +1,175 @@
 /* eslint-disable react/jsx-props-no-spreading */
 import React from 'react';
+import { Formik } from 'formik';
+
 import {
-  Card,
-  Form,
+  SubmitButton,
   Input,
-  Button,
-  DatePicker,
-  Select,
   Checkbox,
-  Row
-} from 'antd';
+  ResetButton,
+  Form,
+  Select,
+  DatePicker,
+  FormItem
+} from 'formik-antd';
+
+import { Card, Button, Row, message, Col } from 'antd';
 
 import { genres } from '../../mock_data';
 
 import { useModal } from '../../hooks';
 
 import 'antd/dist/antd.css';
+import './AddEditMovie.modules.scss';
 
 const { Option } = Select;
 
-const formItemLayout = {
-  labelCol: {
-    xs: {
-      span: 24
-    },
-    sm: {
-      span: 8
-    }
-  },
-  wrapperCol: {
-    xs: {
-      span: 24
-    },
-    sm: {
-      span: 16
-    }
-  }
-};
-const tailFormItemLayout = {
-  wrapperCol: {
-    xs: {
-      span: 24,
-      offset: 0
-    },
-    sm: {
-      span: 16,
-      offset: 8
-    }
-  }
+const validateRequired = (value) => {
+  return value ? undefined : 'required';
 };
 
 export const AddEditMovie = () => {
   const { state } = useModal();
+  const {
+    editContent: { description, image, movieType, name, rating, runtime },
+    id
+  } = state;
   const { openAdd, openEdit, title } = state;
-  const [form] = Form.useForm();
-  const onFinish = (values) => {
-    console.log('Received values of form: ', values);
-  };
 
   const cardWithAddEdit = openAdd || openEdit;
 
+  const deleteMove = () => {
+    console.log(`deleted move ${id}`);
+  };
+
   return (
     <Card title={title} bordered={false} style={{ width: 860 }}>
       {cardWithAddEdit ? (
-        <Form
-          {...formItemLayout}
-          form={form}
-          layout='vertical'
-          name='register'
-          onFinish={onFinish}
-          initialValues={{}}
-          scrollToFirstError
-        >
-          <Row>
-            <Form.Item name='movie-title' label='TITLE'>
-              <Input placeholder='Please enter movie title' />
-            </Form.Item>
+        <Formik
+          initialValues={{
+            title: name,
+            movieUrl: image,
+            overview: description,
+            genre: movieType,
+            rating,
+            runtime
+          }}
+          onSubmit={(values, actions) => {
```

Comment:
As I mentioned previosly you have to keep movies in a state to be able to delete, add, update them, as I understand no one of this operations is not working for now.
You can keep movies in state like:
`const [moviesData, setMoviesData] = useState(movieData);`
or you can event try hook `useReduce` to manage your movies:

![image](https://user-images.githubusercontent.com/52933810/161126398-aad533cd-7d1c-4ec2-a09e-ff4f3d6d7fd6.png)